embed/config.go

```diff
@@ -118,6 +118,7 @@ type Config struct {
 	Metrics               string `json:"metrics"`
 	ListenMetricsUrls     []url.URL
 	ListenMetricsUrlsJSON string `json:"listen-metrics-urls"`
+	ListenMetricsInsecure bool   `json:"listen-metrics-insecure"`
```

Comment:
why is this needed? this should probably be like the listen flags so it accepts "https://whatever,http://whatever" to use insecure connections with http